spec/controllers/plant_populations_controller_spec.rb

```diff
@@ -23,5 +23,15 @@
       expect(json['data'].size).to eq 2
       expect(json['data'].map(&:first)).to match_array pps.map(&:id)
     end
+
+    it 'supports query filtering on ajax request' do
+      pps = create_list(:plant_population, 2).map(&:plant_population_id)
+      get :index, format: :json, query: { plant_population_id: pps[0] }
```

Comment:
Good point - but the test is essentially proper, since it tests response on certain accepted content type. I'll change the wrong name.